vktrace/vktrace_common/vktrace_trace_packet_utils.c

```diff
@@ -213,7 +209,6 @@ vktrace_trace_packet_header* vktrace_create_trace_packet(uint8_t tracer_id, uint
 void vktrace_delete_trace_packet(vktrace_trace_packet_header** ppHeader) {
     vktrace_delete_trace_packet_no_lock(ppHeader);
```

Comment:
Since the vktrace_delete_trace_packet is exactly the same as vktrace_delete_trace_packet_no_lock now, can we remove vktrace_delete_trace_packet_no_lock and rename all its references to vktrace_delete_trace_packet?

Reply:
Yes, vktrace_delete_trace_packet_no_lock can be removed after the change.